src/lib/_common.scss

```diff
@@ -149,9 +149,13 @@ $header-height: 48px;
         align-items: center;
         display: flex;
         flex: 1 1 auto;
-        min-width: 100%;
         position: relative;
     }
+
+    .bp-content.bp-is-fullscreen {
+        background-color: $black;
```

Comment:
I don't think this exists today. Should it only apply to the media player?

Reply:
Oh you're right. Today it just keeps the preview background which is #f5f5f5 I think. But in Safari, the background changed to white. I'll update it with the #f5f5f5 value and make sure it's the same experience across browsers